docs/requirements.txt

```diff
@@ -1,5 +1,5 @@
 -r ../dev-requirements.txt
-sphinx>3.0.0
+sphinx>3.0.0,!=7.2.5
```

Comment:
It looks like something that [is going to be fixed in PyO3](https://github.com/sphinx-doc/sphinx/issues/11662#issuecomment-1713887182). Shouldn't the condition be `<7.2.5` here until then so that the new version of Sphinx does not bring the problem back if PyO3 is still the same version?

Reply:
Sure, good idea!

```suggestion
# https://github.com/sphinx-doc/sphinx/issues/11662#issuecomment-1713887182
sphinx>3.0.0,<7.2.5
```